pb/protos/commands.proto

```diff
@@ -492,6 +492,48 @@ message Rpc {
                 }
             }
         }
+        message SetOrder {
+            message Request {
+                string spaceViewId = 1;
+                repeated string beforeIds = 2;
```

Comment:
Naming is quite confusing here. beforeIds feels like RPC is up to move spaceViewId before this ids.

Better naming can be `precedingIds` or `previousIds`

Reply:
Fixed